e2e/cypress/integration/spatial.spec.js

```diff
@@ -0,0 +1,30 @@
+describe('Spatial', () => {
+    it('Can search geographies by name', () => {
+        cy.request('/api/3/action/location_search?q=california').should((response) => {
+            expect(response.body).to.have.property('success', true);
+            expect(response.body.result[0]).to.have.property('text', 'California');
+        });
+    });
+    it('Can put a package with weird tags in an group', () => {
```

Comment:
What about this introduces weird tags?

Reply:
The `>` signs?

Reply:
That, but also the `/` and the spaces...